CMakeLists.txt

```diff
@@ -6,15 +6,114 @@ project(mlboard
 set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
+# Include modules in the CMake directory.
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/CMake")
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/modules/")
+
 option(BUILD_TESTS "Build tests." ON)
-INCLUDE(FindProtobuf)
-FIND_PACKAGE(Protobuf REQUIRED)
+option(KEEP_LOGS "Keep the files generated during test." OFF)
```

Comment:
You can keep the logs using cmake -DKEEP_LOGS=ON ../

Reply:
I think it's better to name it as -DKEEP_TEST_LOGS. Let me know what you think.